File: src/main/java/de/burrotinto/burroPlayer/interfaces/serial/values/StatusByteConfiguration.java
package de.burrotinto.burroPlayer.interfaces.serial.values;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

/**
 * Created by derduke on 16.02.17.
 */
@Setter
@Getter
@Component
@NoArgsConstructor
@AllArgsConstructor
@ConfigurationProperties(prefix = "burroplayer.adapter.serial.statusByte")
public class StatusByteConfiguration {
    private int playerRunningBit;
    private int playerPausedBit;
}

File: src/main/kotlin/de/burrotinto/burroplayer/interfaces/serial/values/StatusByteConfiguration.kt
package de.burrotinto.burroplayer.interfaces.serial.values

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

/**
 * Created by Florian Klinger on 06.07.17, 21:49.
 */
@Component
@ConfigurationProperties(prefix = "burroplayer.adapter.serial.statusByte")
class StatusByteConfiguration(var playerRunningBit: Int = 0, internal var playerPausedBit: Int = 0)